package.json

```diff
@@ -1,43 +1,52 @@
 {
-  "name": "digitalcheck-apps",
```

Comment:
Keep until renaming (will be renamed when renaming the repo I guess?) or change to "digitalcheck-dito" as discussed.